<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-FDHEC2G" failures="5" tests="24" name="Regression test" time="0.107" errors="0" timestamp="2023-04-16T12:29:43 EET">
  <testcase classname="test.Class1" name="Class1test1" time="0.017">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Class1.Class1test1(Class1.java:9)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- Class1test1 -->
  <testcase classname="test.Class1" name="Class1test2" time="0.001"/>
  <testcase classname="test.Class1" name="Class1test3" time="0.001"/>
  <testcase classname="test.Class1" name="Class1test4" time="0.0"/>
  <testcase classname="test.Class2" name="Class2test1" time="0.001"/>
  <testcase classname="test.Class2" name="Class2test2" time="0.0"/>
  <testcase classname="test.Class2" name="Class2test3" time="0.001"/>
  <testcase classname="test.Class2" name="Class2test4" time="0.0"/>
  <testcase classname="test.Class3" name="Class3est3" time="0.001"/>
  <testcase classname="test.Class3" name="Class3test1" time="0.001"/>
  <testcase classname="test.Class3" name="Class3test2" time="0.001"/>
  <testcase classname="test.Class3" name="Class3test4" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Class3.Class3test4(Class3.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- Class3test4 -->
  <testcase classname="test.Class4" name="Class4est3" time="0.0"/>
  <testcase classname="test.Class4" name="Class4test1" time="0.0"/>
  <testcase classname="test.Class4" name="Class4test2" time="0.001"/>
  <testcase classname="test.Class4" name="Class4test4" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Class4.Class4test4(Class4.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- Class4test4 -->
  <testcase classname="test.Class5" name="Class5est3" time="0.001"/>
  <testcase classname="test.Class5" name="Class5test1" time="0.001"/>
  <testcase classname="test.Class5" name="Class5test2" time="0.001"/>
  <testcase classname="test.Class5" name="Class5test4" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Class5.Class5test4(Class5.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- Class5test4 -->
  <testcase classname="test.Class6" name="Class6est3" time="0.001"/>
  <testcase classname="test.Class6" name="Class6test1" time="0.001"/>
  <testcase classname="test.Class6" name="Class6test2" time="0.001"/>
  <testcase classname="test.Class6" name="Class6test4" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Class6.Class6test4(Class6.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- Class6test4 -->
</testsuite> <!-- Regression test -->
